Compile error

Version 0.3.2 fails to build on ArchLinux. Any ideas?

```
-- The C compiler identification is GNU 7.2.0
-- The CXX compiler identification is GNU 7.2.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2") 
-- Checking for modules 'polkit-gobject-1;gthread-2.0;gtk+-3.0;switchboard-2.0;granite;gnome-settings-daemon'
--   Found polkit-gobject-1, version 0.114
--   Found gthread-2.0, version 2.52.3
--   Found gtk+-3.0, version 3.22.21
--   Found switchboard-2.0, version 2.0
--   Found granite, version 0.4.1
--   Found gnome-settings-daemon, version 3.24.3
-- Found Vala: /usr/bin/valac  
-- checking for Vala version of 0.34.1
--   found Vala, version 0.36.4
-- Checking for modules 'glib-2.0;gio-2.0;polkit-gobject-1'
--   Found glib-2.0, version 2.52.3
--   Found gio-2.0, version 2.52.3
--   Found polkit-gobject-1, version 0.114
-- Configuring done
-- Generating done
-- Build files have been written to: /build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/build
[  5%] Generating main.c;Server.c;../src/Interfaces.c
[ 11%] Generating Plug.c;Battery.c;Interfaces.c;PowerSupply.c;ActionComboBox.c;LidCloseActionComboBox.c;TimeoutComboBox.c;Permission.c;LogindHelper.c;/build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/build/src/config.c
Scanning dependencies of target i18n
/build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/cli/../src/Interfaces.vala:56.9-56.36: warning: method `Power.UpowerDevice.refresh' never used
        public abstract void refresh () throws IOError;
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/cli/../src/Interfaces.vala:70.9-70.54: warning: method `Power.Upower.enumerate_devices' never used
        public abstract ObjectPath[] enumerate_devices () throws IOError;
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Compilation succeeded - 2 warning(s)
Scanning dependencies of target logind-helper
[ 17%] Building C object cli/CMakeFiles/logind-helper.dir/main.c.o
[ 23%] Building C object cli/CMakeFiles/logind-helper.dir/Server.c.o
[ 29%] Building C object cli/CMakeFiles/logind-helper.dir/__/src/Interfaces.c.o
cc: error: /build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/build/cli/main.c: No such file or directory
cc: fatal error: no input files
compilation terminated.
make[2]: *** [cli/CMakeFiles/logind-helper.dir/build.make:79: cli/CMakeFiles/logind-helper.dir/main.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
cc: error: /build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/build/src/Interfaces.c: No such file or directory
cc: fatal error: no input files
compilation terminated.
make[2]: *** [cli/CMakeFiles/logind-helper.dir/build.make:127: cli/CMakeFiles/logind-helper.dir/__/src/Interfaces.c.o] Error 1
/build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/src/Plug.vala:146.67-146.102: warning: copying delegates is not supported
/build/switchboard-plug-power/src/switchboard-plug-power-0.3.2/src/Plug.vala:146.105-146.140: warning: copying delegates is not supported
Compilation succeeded - 2 warning(s)
Scanning dependencies of target pantheon-power
[ 52%] Building C object src/CMakeFiles/pantheon-power.dir/Plug.c.o
[ 52%] Building C object src/CMakeFiles/pantheon-power.dir/Interfaces.c.o
[ 52%] Building C object src/CMakeFiles/pantheon-power.dir/Battery.c.o
[ 52%] Building C object src/CMakeFiles/pantheon-power.dir/PowerSupply.c.o
[ 58%] Building C object src/CMakeFiles/pantheon-power.dir/ActionComboBox.c.o
[ 64%] Building C object src/CMakeFiles/pantheon-power.dir/LidCloseActionComboBox.c.o
make[1]: *** [CMakeFiles/Makefile2:262: cli/CMakeFiles/logind-helper.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 70%] Building C object src/CMakeFiles/pantheon-power.dir/TimeoutComboBox.c.o
[ 76%] Building C object src/CMakeFiles/pantheon-power.dir/Permission.c.o
[ 82%] Building C object src/CMakeFiles/pantheon-power.dir/LogindHelper.c.o
[ 88%] Building C object src/CMakeFiles/pantheon-power.dir/config.c.o
[ 88%] Built target i18n
[ 94%] Linking C shared module libpantheon-power.so
[ 94%] Built target pantheon-power
make: *** [Makefile:130: all] Error 2
==> ERROR: A failure occurred in build().
```

--- a/cli/CMakeLists.txt
+++ b/cli/CMakeLists.txt
@@ -13,7 +13,7 @@ configure_file (io.elementary.logind.helper.service.in ${CMAKE_CURRENT_BINARY_DI
 vala_precompile (CLI_VALA_C ${CLI_EXEC_NAME}
     main.vala
     Server.vala
-    ../src/Interfaces.vala
+    ${CMAKE_SOURCE_DIR}/src/Interfaces.vala
 PACKAGES
     glib-2.0
     gio-2.0

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -31,7 +31,7 @@ PACKAGES
     polkit-gobject-1
     posix
 OPTIONS
-    --vapidir=${CMAKE_CURRENT_SOURCE_DIR}/../vapi/
+    --vapidir=${CMAKE_SOURCE_DIR}/vapi/
     --thread
 	${VALA_DEFINED_VAR}
 )

